### PR TITLE
Add all sites AN export to TSV

### PR DESCRIPTION
**Pull request by jkgoodrich**
You can see the genomes TSV output here: gs://gnomad/release/4.1/tsv/genomes/gnomad.genomes.v4.1.allele_number.tsv.bgz

### CODE REVIEW

File: gnomad_qc/v4/resources/release.py [hunk 1]
```diff
@@ -329,19 +329,36 @@ def release_coverage_tsv_path(
...
+    """
+    Fetch path to all sites AN TSV file.
+
+    :param data_type: 'exomes' or 'genomes'. Default is 'exomes'.
+    :param release_version: Release version. Default is
+        CURRENT_ALL_SITES_AN_RELEASE["exomes"].
+    :param test: Whether to use a tmp path for testing. Default is False.
+    :return: All sites AN TSV path.
+    """
+    return (
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/resources/release.py:360]: MAybe assign release version in the function? Default to None on L348, then sometting like
```suggestion
    release_version = release_version if release_version is not None else CURRENT_ALL_SITES_AN_RELEASE[data_type]
    return (
```

File: gnomad_qc/v4/annotations/compute_coverage.py [hunk 2]
```diff
@@ -312,13 +313,21 @@ def get_coverage_resources(
         },
         pipeline_input_steps=[compute_coverage_ht],
     )
+    export_an_tsv = PipelineStepResourceCollection(
+        "--export-all-sites-an-release-tsv",
+        output_resources={
+            "allele_number_tsv": release_all_sites_an_tsv_path(data_type, test=test)
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/annotations/compute_coverage.py:319]: Wont the release version always be the `CURRENT_ALL_SITES_AN_VERSION["exomes"]`. I know they are in sync now but should probably adjust the function call or function itself?

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]
Just one thing on the tsv path call/function

File: gnomad_qc/v4/annotations/compute_coverage.py (hunk 2 above)
Comment (jkgoodrich) [reply gnomad_qc/v4/annotations/compute_coverage.py:319]: yup, thanks for catching. I fixed this for the other functions in my combined freq PR, but forgot to change it here

**Review by mike-w-wilson — APPROVED** [review]

**Pull request merged**